CHANGELOG.md

```diff
@@ -1,2359 +1,2803 @@
+
+    ### Additions
+
+    -   Add support for `custom_unit_amount` on parameter class `stripe.Product.CreateParamsDefaultPriceData`
+    -   Add support for `usage_threshold` on parameter class `stripe.billing.Alert.CreateParams` and resource `stripe.billing.Alert`
+    -   Add support for `allow_redisplay` on parameter classes `stripe.terminal.Reader.ProcessPaymentIntentParamsProcessConfig` and `stripe.terminal.Reader.ProcessSetupIntentParams`
+    -   Add support for `international_transaction` on enum `stripe.treasury.ReceivedCredit.failure_code`
+    -   Add support for `2024-09-30.acacia` on enum `stripe.WebhookEndpoint.CreateParams.api_version`
+    -   Add support for new Usage Billing APIs `stripe.v2.billing.MeterEvent`, `stripe.v2.billing.MeterEventAdjustments`, `stripe.v2.billing.MeterEventSession`, `stripe.v2.billing.MeterEventStream` and the new Events API `stripe.v2.core.Events` under the [v2 namespace ](https://docs.corp.stripe.com/api-v2-overview)
+    -   Add method [rawRequest()](https://github.com/stripe/stripe-python/tree/master?tab=readme-ov-file#custom-requests) on the `StripeClient` class that takes a HTTP method type, url and relevant parameters to make requests to the Stripe API that are not yet supported in the SDK.
+
+    ### Other changes
+
+    -   Change type of `default_allowed_updates` on `stripe.billing_portal.Configuration.CreateParamsFeaturesSubscriptionUpdate` from `Union[Literal[''], List[Literal['price', 'promotion_code', 'quantity']]]` to `NotRequired[Literal['']|List[Literal['price', 'promotion_code', 'quantity']]]`
+    -   Change type of `products` on `stripe.billing_portal.Configuration.CreateParamsFeaturesSubscriptionUpdate` from `Union[Literal[''], List[Configuration.CreateParamsFeaturesSubscriptionUpdateProduct]]` to `NotRequired[Literal['']|List[Configuration.CreateParamsFeaturesSubscriptionUpdateProduct]]`
+
 ## 10.13.0b1 - 2024-09-18
-* [#1395](https://github.com/stripe/stripe-python/pull/1395) Update generated code for beta
-  * Add support for `send_money` on parameter class `stripe.AccountSession.CreateParamsComponentsFinancialAccountFeatures`
-  * Add support for `transfer_balance` on parameter class `stripe.AccountSession.CreateParamsComponentsFinancialAccountFeatures`
-  * Add support for `automatically_finalizes_at` on resource `stripe.QuotePreviewInvoice`
-  * Remove support for resource `stripe.QuotePhase`
-  * Add support for `rechnung` on enums `stripe.PaymentLink.payment_method_types`, `stripe.PaymentLink.CreateParams.payment_method_types`, and `stripe.PaymentLink.ModifyParams.payment_method_types`
-  * Add support for `terminal_reader_invalid_location_for_activation` on enum `stripe.QuotePreviewInvoice.LastFinalizationError.code`
+
+-   [#1395](https://github.com/stripe/stripe-python/pull/1395) Update generated code for beta
```

Comment:
Can we avoid all these formatting changes?

Reply:
that's bizarre...we definitely should be able to.  let me take a look

Reply:
I did the best I could.  there are about 17 lines that github is _convinced_ have been modified and I can't seem to convince it otherwise, but I got the bulk of the formatting changes reverted.  I also checked in changes to .editorconfig and vscode settings.json to turn off auto formatting on save for changelog.md (and for vscode, markdown files in general)